canary/producer-cpp/jobs/orchestrator.groovy

```diff
@@ -0,0 +1,217 @@
+import jenkins.model.*
+
+WORKSPACE_PRODUCER="src"
+// WORKSPACE_CONSUMER="consumer-java"
+GIT_URL='https://github.com/aws-samples/amazon-kinesis-video-streams-demos.git'
+GIT_HASH='new-canary-producer-cpp'
```

Comment:
Remember to change this to master before merging because the branch automatically gets deleted once the PR is merged.